cli/analysis/analysis.go

```diff
@@ -0,0 +1,193 @@
+// Copyright © 2019 Globo.com
```

Comment:
We have added the following on all other files of huskyCI: 
```
// Copyright 2019 Globo.com authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.
```
Do you think we could use this template or do you prefer using this new one you are proposing?

Reply:
This is the default for the cobra package. Setting `~/.cobra.yaml` as following and using `cobra add login`, this generate template files for new commands.
```
author: Globo.com
license: 3-Clause BSD
```